src/molecule.py

```diff
@@ -489,8 +456,8 @@ def AStr(self):
             return ','.join(['%i' % i for i in self.L()])
         def e(self):
             """ Return an array of the elements.  For instance ['H' 'C' 'C' 'H']. """
-            elems = nx.get_node_attributes(self,'e')
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.